packages/fxa-settings/src/lib/logger.test.js

```diff
@@ -0,0 +1,162 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { assert } from 'chai';
+import sinon from 'sinon';
+import Logger from './logger';
+
+describe('lib/logger', function() {
+  var logger;
```

Comment:
Is there any reason these are `var`s instead of `let`s that you know of? (I know it was probably just a copy and paste)